mne/preprocessing/tests/test_maxwell.py

```diff
@@ -0,0 +1,84 @@
+# Author: Mark Wronkiewicz <wronk@uw.edu>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import warnings
+from mne.datasets import sample
+from mne.io import Raw
+from mne.datasets import testing
+
+from numpy.testing import (assert_equal, assert_allclose,
+                           assert_array_almost_equal)
+import numpy as np
+
+from mne.preprocessing import maxwell
+
+from scipy.special import sph_harm as scipy_sph_harm
+
+warnings.simplefilter('always')  # Always throw warnings
+
+
+def test_spherical_harmonic():
+    """Test for spherical harmonics"""
+    deg = 1
+    order = -1
+    azimuth = np.random.random_sample(size=(50, 1)) * 2 * np.pi
+    polar = np.random.random_sample(size=(50, 1)) * np.pi
+
+    # Internal calculation
+    sph_harmonic = maxwell._sph_harmonic(deg, order, azimuth, polar)
+    # Check against scipy
+    sph_harmonic_scipy = scipy_sph_harm(order, deg, azimuth, polar)
+
+    assert_array_almost_equal(sph_harmonic, sph_harmonic_scipy, decimal=15,
+                              err_msg='Spherical harmonic calculation mismatch')
+
+
+@testing.requires_testing_data
+def test_maxwell_filter():
+    """Test multipolar moment and Maxwell filter"""
+
+    # Load data
+    data_path = sample.data_path()
+    raw_fname = op.join(data_path, 'MEG/sample', 'sample_audvis_raw.fif')
+    raw = Raw(raw_fname, preload=False, proj=False).crop(0., 1., False)
+    raw.preload_data()
+
+    #sss_proc_fname = op.join(data_path, 'sample_audvis_raw_sss.fif')
+    #sss_benchmark = mne.io.Raw(sss_proc_fname, preload=False,
+    #                           proj=False).crop(0., 1., False)
+    #sss_benchmark.preload_data()
```

Comment:
I have to wait until the maxwell filter is working. Then I'll compute the benchmark maxwell filter dataset, upload it, and incorporate it into the tests.